buf.yaml

```diff
@@ -3,7 +3,7 @@ deps:
   - buf.build/googleapis/googleapis
 breaking:
   use:
-    - FILE
+    - WIRE
```

Comment:
just curious what does this do?

Reply:
Ohh, this is temporary, since its a backward incompatible change, this fails the build. 
The plan is to reset this to FILE when this PR is merged.